bridge/bridge/WorkflowUtils.py

```diff
@@ -100,3 +105,44 @@ def create_conversion_rate_calculator_factory(execution_context, databases, nati
 	return ConversionRateCalculatorFactory(databases, native_facade.extract_mosaic_id().formatted, execution_context.is_unwrap_mode)
 
 # endregion
+
+
+# region prepare_send
+
+def prepare_send(network, request, conversion_function, fee_multiplier):
+	"""Performs basic calculations and validation prior to sending a payout transaction."""
+
+	def make_error(error_message):
+		return PrepareSendResult(error_message, None, None)
+
+	if not fee_multiplier:
+		fee_multiplier = Decimal('1')
+	else:
+		fee_multiplier *= Decimal(conversion_function(10 ** 12)) / Decimal(10 ** 12)
```

Comment:
Worth checking fee_multiplier is >= 0 ? On negative values we might get unexpected results.